src/styles/styles.less

```
@@ -623,6 +623,16 @@ hr {
   word-break: normal;
 }
 
+// roll-graphic.hbs
+.dice-formula.roll-graphic {
+  padding: 0.25rem;
+  gap: 0.25rem;
+  display: flex;
+  margin: 0;
+  justify-content: center;
+  flex-direction: row;
+}
+
```

Could we do this as well? It would make the preroll dialog look nice again. 😄 

```suggestion

fieldset {
  padding: 0;
  border: none;
}
```